### ISSUE TITLE
v1.0 doesn't seem to reflect the decision about the various single-attribute tables

### ISSUE THREAD

**Issue by monfresh** [openreferral/specification#78]
Back in December, I had proposed that the tables that only represent a single attribute should instead be fields within each table they belong to. The fields that represent an array of short String items should use a comma-separated list. The tables are: accessibility, accreditations, licenses, funding_sources, service_areas, eligibility, required_documents, fees, accepted_payments, interpretation_services, and languages. 

According to the Open Referral Assembly video session we recorded in December, [@spara agreed with me](https://www.youtube.com/watch?v=1gTl7og0Sps&t=1178), so I assumed this change would be made. You can hear Sophia discuss this between 19:40 and 20:20.

Also, I think `fees` and `interpretation_services` made it on this list by mistake since they're meant to be descriptions rather than a list of unique items. For example, the `fees` attribute in San Mateo County data mostly consists of sentences like "Membership fee and sliding scale. A 1-day free guest pass is available so prospective members may try out the facilities."

As for `interpretation_services`, it would be used to describe what kind of services are provided, such as "We offer 3-way interpretation calls via a third party called XYZ (with a link to the XYZ website). We also have a native Spanish-speaker on premises on Wednesdays".

**Comment by NeilMcKechnie** [comment]
There are several issues in this post which should be broken into separate items. I will just comment on the first one.

I agree the model would be alot simpler if these single-field attributes could be collapsed into their base object. It is pretty rare from a search or display perspective to have those data be normalized. So I support moving the following to their parent object: Accessibility, accreditations, licenses, funding sources, fees and interpretation_services.

However service_areas needs to stay normalized and in fact i will submit a new request that they be better structured. Also I think the following fields should remain normalized in their own child objects: eligibility, required documents, accepted_payments and languages. These can be important factors to use when searching/filtering data and as such would benefit from the higher structure that comes with normalization.

**Comment by timgdavies** [comment]
Tagging the proposal here for discussion in 1.1 (after we've completed tidy up of current spec). 

To summarise a proposal

## Tables that that should become properties of service

| Field | Definition | Type | Values |
|------|-----------|------|--------|
| accessibility | Description of assistance or infrastructure that facilitate access to clients with disabilities. | comma separated list |  "cd", "deaf_interpreter", "disabled_parking", "elevator", "ramp", "restroom", "tape_braille", "tty", "wheelchair", "wheelchair_van" |  
| fees | A listing of the costs of services, including free ones | free text | free text |
| interpretation_services | Languages, other than English, for which interpretation is available. Languages are listed as ISO639-1 codes. | free text | free text |

New fields, not currently covered in the datapackage.json (covered in the common vocabulary table, but not actually in the spec...)

| Field | Definition | Type | Values |
|------|-----------|------|--------|
| accreditations | Accreditation is the formal evaluation of an organization or program against best practice standards set by an accrediting organization. | free text | N/A |
| licenses | An organization may have a license issued by a government entity to operate legally. | comma separate list | short free text values |


## Kept as normalised tables

| Field | Definition | Type | Values |
|------|-----------|------|--------|
| funding_sources | Source of funds for organization or service | free text | free text |
| service_areas | The geographic area where a service is available. This is a free-text description, and so may be precise or indefinite as necessary. | free text  | See #95 for discussion | 
| eligibility | The rules or guidelines that determine who can receive the service. | enum list | "adult","child","teen","family","female","male", "Transgender", "Transgender - M to F", "Transgender - F to M" |
| required_documents | The document required to apply for or receive the service. e.g. 'Government-issued ID', 'EU Passport' | free text | N/A | 
| payment_accepted |  The methods of payment accepted for the service | enum list | "cash", "check", "money order", "credit card", "medicare", "SNAP", "WIC", "EBT" |
| languages | Languages, other than English, in which the service is delivered. Languages are listed as ISO639-1 codes. | codelist | en, es etc. |


* I've kept funding_sources in the list, because the current model allows this to link to either an organisation or a service. I can see use-cases for this as a normalised value. 
* Some of the lists (e.g. for eligibility) may also need to be consulted on and updated
* See #77 for discussion of naming of payment_accepted field/table

**Comment by klambacher** [comment]
The challenge of collapsing these elements into a a single table is you eliminate the ability for extensions for those of us that keep extra or detailed data. Obviously the common ground is extremely important for wide exchange, but extension is equally important for adoption.

For example, we keep accessibility and fee information normalized (using a set of standard "checklist" values) with additional notes available to describe the normalized data. This is especially important for accessibility, where information is useless without detail. Some types of disabilities may require pre-planning of seemingly mundane details - there is a big difference between an "accessible washroom" with an automatic door, and one with a wheelchair stall and no automatic door. The ability to quickly identify the accessible washroom (through a standardized, searchable value) is important, and the need for that extra detail (do I need a companion with me?) is also important. Not all systems incorporate this extra detail, but unless you structure the data in way that allows it to be stored and shared effectively for searching and viewing, there is a potential problem. 

To summarize - based on timgdavies proposal above, I'd definitely like to see everything list in "keep as normalized tables" to stay that way, and would prefer that accessibility at least also be normalized. Though we keep normalized fee tables, it is less essential and more a data entry convenience.

**Comment by NeilMcKechnie** [comment]
I am supportive of @klambacher 's points. Also accreditations and licenses could probably safely live only at the organization level, not service level.
